core/runner/pom.xml

```diff
@@ -80,6 +80,18 @@
     <dependency>
       <groupId>org.apache.sling</groupId>
       <artifactId>org.apache.sling.commons.osgi</artifactId>
+      </dependency>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>org.osgi.service.component.annotations</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>org.osgi.annotation</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
Can we now remove `org.apache.felix.scr` annotation dependencies now from Runner's `pom.xml`?

Reply:
Unfortunatelly, there is some dependency which blocked it. We should investigate it, but now I don`t know which dependency it is :/

Reply:
I have some propositions for this pull request that allow to remove these dependencies from pom:
https://github.com/Cognifide/aet/pull/309